### ISSUE TITLE
Technical Problem idea!

### ISSUE THREAD

**Issue by Bertogonz3000** [4ndyNMC/MetaU-Capstone#16]
https://github.com/4ndyNMC/MetaU-Capstone/blob/main/app/src/main/java/com/example/metaucapstone/Recipe.java#L43

Potential new complex technical problem: consider finding a way to have this list auto update with the API!

EX: Once a day or once a week the app will read all cuisines from the API and update a text document stored on the phone or in your database. On loading in the app, this map is created from that text document. That way you'll never need to worry about something like this going wrong.  This could be expanded to several different areas in this app, and I think it would be a really solid new complex technical problem. Consider how wide reaching this could be. I'm actually sort of excited by this idea.

**Comment by Bertogonz3000** [comment]
Of course, this assumes you'll be able to query all possible cuisines and that all this would work. Dealing with that kind of ambiguity is what would make this a complex technical problem and is exactly the kind of behavior we would love to see.

**Comment by 4ndyNMC** [comment]
I remember you suggesting this to me in one of our previous 1:1s! I'll keep this in mind for later.
